src/main/java/com/amazon/opendistroforelasticsearch/sql/antlr/parser/OpenDistroSqlParser.java

```diff
@@ -0,0 +1,7485 @@
+			if ( listener instanceof OpenDistroSqlParserListener ) ((OpenDistroSqlParserListener)listener).exitDeleteStatement(this);
+		}
+		@Override
+		public <T> T accept(ParseTreeVisitor<? extends T> visitor) {
+			if ( visitor instanceof OpenDistroSqlParserVisitor ) return ((OpenDistroSqlParserVisitor<? extends T>)visitor).visitDeleteStatement(this);
+			else return visitor.visitChildren(this);
+		}
+	}
+
+	public final DeleteStatementContext deleteStatement() throws RecognitionException {
+		DeleteStatementContext _localctx = new DeleteStatementContext(_ctx, getState());
+		enterRule(_localctx, 8, RULE_deleteStatement);
+		try {
+			enterOuterAlt(_localctx, 1);
+			{
+			setState(162);
+			singleDeleteStatement();
+			}
+		}
+		catch (RecognitionException re) {
+			_localctx.exception = re;
+			_errHandler.reportError(this, re);
+			_errHandler.recover(this, re);
+		}
+		finally {
+			exitRule();
+		}
+		return _localctx;
+	}
+
+	public static class SingleDeleteStatementContext extends ParserRuleContext {
+		public TerminalNode DELETE() { return getToken(OpenDistroSqlParser.DELETE, 0); }
+		public TerminalNode FROM() { return getToken(OpenDistroSqlParser.FROM, 0); }
+		public TableNameContext tableName() {
+			return getRuleContext(TableNameContext.class,0);
+		}
+		public TerminalNode WHERE() { return getToken(OpenDistroSqlParser.WHERE, 0); }
+		public ExpressionContext expression() {
+			return getRuleContext(ExpressionContext.class,0);
+		}
+		public OrderByClauseContext orderByClause() {
+			return getRuleContext(OrderByClauseContext.class,0);
+		}
+		public TerminalNode LIMIT() { return getToken(OpenDistroSqlParser.LIMIT, 0); }
+		public DecimalLiteralContext decimalLiteral() {
+			return getRuleContext(DecimalLiteralContext.class,0);
+		}
+		public SingleDeleteStatementContext(ParserRuleContext parent, int invokingState) {
+			super(parent, invokingState);
+		}
+		@Override public int getRuleIndex() { return RULE_singleDeleteStatement; }
+		@Override
+		public void enterRule(ParseTreeListener listener) {
+			if ( listener instanceof OpenDistroSqlParserListener ) ((OpenDistroSqlParserListener)listener).enterSingleDeleteStatement(this);
+		}
+		@Override
+		public void exitRule(ParseTreeListener listener) {
+			if ( listener instanceof OpenDistroSqlParserListener ) ((OpenDistroSqlParserListener)listener).exitSingleDeleteStatement(this);
+		}
+		@Override
+		public <T> T accept(ParseTreeVisitor<? extends T> visitor) {
+			if ( visitor instanceof OpenDistroSqlParserVisitor ) return ((OpenDistroSqlParserVisitor<? extends T>)visitor).visitSingleDeleteStatement(this);
+			else return visitor.visitChildren(this);
+		}
+	}
+
+	public final SingleDeleteStatementContext singleDeleteStatement() throws RecognitionException {
+		SingleDeleteStatementContext _localctx = new SingleDeleteStatementContext(_ctx, getState());
+		enterRule(_localctx, 10, RULE_singleDeleteStatement);
+		int _la;
+		try {
+			enterOuterAlt(_localctx, 1);
+			{
+			setState(164);
```

Comment:
should the statement in the bracket be lined up with the bracket itself?

Reply:
Same as above.